classes/patreon_wordpress.php

```diff
@@ -219,15 +319,49 @@ public static function getUserPatronage() {
 		return false;
 
 	}
-
+	
 	public static function getUserPatronageDuration($pledge) {
 
 		$user_response = self::getPatreonUser($user);
 
 		$patronage_age = 0;
 
 	}
+	
+	public static function checkDeclinedPatronage($user) {
+		
+		if(!$user)
+		{
```

Comment:
format ifs

Reply:
Committed:

https://github.com/codebard/patreon-wordpress/commit/a25c142224c1f28442c847e76185cb08420cb059

Reply:
seems like that commit does not address this